testsuite/features/support/constants.rb

```diff
@@ -461,3 +461,17 @@
     SLE-Module-Desktop-Applications15-SP4-Updates
   ]
 }.freeze
+
+# MAC addresses for the nested VM from galaxy-infrastructure for the different pipelines
+NESTED_VM_MACS = { 'uyuni-master-min-nested' => 'aa:b2:93:01:00:df',
+                   'suma-head-min-nested'    => 'aa:b2:93:01:00:bf',
+                   'suma-pr1-min-nested'     => 'aa:b2:92:04:00:0b',
+                   'suma-pr2-min-nested'     => 'aa:b2:92:04:00:1b',
+                   'suma-pr3-min-nested'     => 'aa:b2:92:04:00:2b',
+                   'suma-pr4-min-nested'     => 'aa:b2:92:04:00:3b',
+                   'suma-pr5-min-nested'     => 'aa:b2:92:04:00:4b',
+                   'suma-pr6-min-nested'     => 'aa:b2:92:04:00:5b',
+                   'suma-pr7-min-nested'     => 'aa:b2:92:04:00:6b',
+                   'suma-pr8-min-nested'     => 'aa:b2:92:04:00:7b',
+                   'suma-pr9-min-nested'     => 'aa:b2:92:04:00:8b',
+                   'suma-pr10-min-nested'    => 'aa:b2:92:04:00:9b' }.freeze
```

Comment:
Ideally, we should not include internal infrastructure information in our test framework.
I would consider the possibility of moving this information to environment variables inside the .bashrc file on the controller, at sumaform side.

https://github.com/uyuni-project/sumaform/blob/master/salt/controller/bashrc

Passing the necessary information through the main.tf

Reply:
We indeed do it that way for the PXE boot minion:
```
suma-42-ctl:~ # env | grep PXEBOOT_MAC
PXEBOOT_MAC=52:54:00:26:02:BA
```
Please note that bash has some support for arrays IIRC. That should allow avoiding to multiply the variables. Please check that you can then extract these from ruby before you start...

Reply:
I moved the MAC addresses to sumafom and the .bashrc and am currently testing those changes. I would request another review after they are successful.